Packs/DragosWorldview/Integrations/DragosWorldview/DragosWorldview.py

```diff
@@ -0,0 +1,212 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+"""Dragos Worldview Integration for XSOAR."""
+import json
+from datetime import datetime, timedelta
+from typing import Any, Dict, Tuple
+
+import dateparser
+import requests
+
+# flake8: noqa: F402,F405 lgtm
+
+
+STATUS_TO_RETRY = [500, 501, 502, 503, 504]
+
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint:disable=no-member
+
+
+class Client(BaseClient):
+    def whoami(self) -> Dict[str, Any]:
+        """Test that API works."""
+        return self._http_request(
+            method="get",
+            url_suffix="indicators?page=1&page_size=1&serial%5B%5D=AA-2022-24",
```

Comment:
What are the hardcoded value here `serial%5B%5D=AA-2022-24`?

Reply:
The hardcoded value there is a query that looks for indicators associated with a specific report of serial number AA-2022-24.

Reply:
and this value are constant and common or something related to specific user?

Reply:
These are constant values not specific to the user